import-dashboards.py

```diff
@@ -229,7 +229,10 @@ def add_panels():
         if os.path.isdir(source_dir):
             files_list = os.listdir(source_dir)
             print '  * Copying %r' % (app,)
-            shutil.rmtree(dest_dir, True)
+            try:
+                shutil.rmtree(dest_dir, False)
+            except Exception as err:
+                print '  * Failed to remove %s: %s' % (dest_dir, err)
```

Comment:
And what about to add `except` here https://github.com/percona/pmm-server/pull/196/files#diff-b473c5eafebc34251b1785f042d44e4eR432 - to log any other error?